scripts/Install-Extension.ps1

```diff
@@ -0,0 +1,249 @@
+<#
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+.SYNOPSIS
+This script provides helpers to install or uninstall ebpf extensions.
+
+.PARAMETER Extension
+    Specifies the extension to install. This MUST be one of ("neteventebpfext", "ntosebpfext").
+
+.PARAMETER Action
+    Specifies the action to take. This MUST be either "Install" or "Uninstall".
+
+.PARAMETER BinaryDirectory
+    Specifies the directory containing the necessary binaries().
+    This MUST be the full path to the directory. By default, the current directory is used.
+
+.EXAMPLE
+    Install_Extension.ps1 -Action Install
+
+.EXAMPLE
+    Install_Extension.ps1 -Action Uninstall -Extension "neteventebpfext" 
+
+.EXAMPLE
+    Install_Extension.ps1 -Action Install -BinaryDirectory "C:\binaries"
+#>
+
+param (
+    [Parameter(Mandatory=$true)]
+    [ValidateSet("Install", "Uninstall")]
+    [string]$Action,
+    [Parameter(Mandatory=$true)]
+    [ValidateSet("neteventebpfext", "ntosebpfext")]
+    [string]$Extension,
+    [Parameter(Mandatory=$false)]
+    [string]$BinaryDirectory = (Get-Location).Path
+)
+
+Set-StrictMode -Version 'Latest'
+$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
+
+<#
+.SYNOPSIS
+Stops and deletes a service.
+
+.PARAMETER ServiceName
+The name of the service to cleanup.
+
+.EXAMPLE
+Clear-Service -ServiceName "neteventebpfext"
+#>
+function Clear-Service(
+    [Parameter(Mandatory=$true)]
+    [string]$ServiceName
+) {
+    # Wait for the service to stop.
+    $StopSuccess = $false
+    try {
+        Stop-Service $ServiceName
+        for ($i = 0; $i -lt 100; $i++) {
```

Comment:
(not blocking): Do you just want to do something straightforward like 10 retires of 1 second each? I think just looking at a larger number of miliseconds and iterations to do a bit more mental math is less readable.